src/databases/airtable.database.ts

```
@@ -0,0 +1,1073 @@
+							break
+
+						case DatabaseColumnType.BOOLEAN: 
+							options = {
+								icon: "check",
+								color: "grayBright" 
+							}
+							break
+
+						case DatabaseColumnType.DATE:
+
+							let timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone ?? 'client'
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)